### DISCUSSION TITLE
Switching languages is not supported on GitHub mobile

### DISCUSSION THREAD

**Discussion by sy-records** [community/community#25] [Mobile] [answered]
I'm using IOS14 and when I choose to switch languages in the app, it always jumps to the system settings, but there is no language option.

**Comment by candyho** [comment, marked as answer]
Hey @sy-records , in order to switch languages in the GitHub iOS app, you'd need to add the preferred languages at the system level first. 

To do that:
1. close GitHub mobile app 
2. open "Settings" > "Language & Region" > "Add Language" under "Preferred Language Order"
3. add your preferred languages
4. when you're done adding the languages, open GitHub mobile app again 
5. go to in-app "Settings" > "App Language" > "Preferred Languages" 
6. you should be able to pick the languages you just added to the system.

**Reply by sy-records** [reply]
OK, thank you.

**Reply by alexcnichols** [reply]
Might make sense to add more about this to the [documentation](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/github-for-mobile#supported-languages-for-github-for-mobile), including a link to the Apple docs about adding a language to the system.

**Reply by ghost** [reply]
Sorry for the necrobump: I've already tried it but seems not to accept Italian as valid language. It points to GH's settings but does not show any valid language
